com.github.gmg137.netease-cloud-music-gtk.yml

```diff
@@ -0,0 +1,46 @@
+app-id: com.github.gmg137.netease-cloud-music-gtk
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: netease-cloud-music-gtk
+rename-icon: netease-cloud-music-gtk
+rename-desktop-file: netease-cloud-music-gtk.desktop
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-music
+  - --persist=.lyrics
+  - --metadata=X-DConf=migrate-path=/com/github/gmg137/netease-cloud-music-gtk/
```

Comment:
Is it actually needed, or just keeping it here for the sake of pre-flatpak installs?

Reply:
Removed

Reply:
Removed